script/Arcueid/Day 1/01_00_ARC01_2_1.txt

```diff
@@ -276,7 +276,7 @@ I got along well with the people of the Arima family, and I treated my stepmothe
 [sha:7284d137dd01ed90917bc54d01d2117bef587621]{
 -- Page 155, Offset 1564.
 -- もともと一般的な家庭に憧れていたところもあって、遠野志貴は有間の家で、本当の子供のように暮らしてきた。
-Tohno Shiki has always longed for a normal family,
+Tohno Shiki had always longed for a normal family,
  and lived with the Arimas as though he were their actual son.
```

Comment:
I think I recall you saying at some point that the notes say he refers to himself in third person a lot, so that might be why some instances are like that. Happy to remove it though.